.eslintrc.yml

```diff
@@ -1,6 +1,10 @@
-extends: ["@redhat-cloud-services/eslint-config-redhat-cloud-services", "plugin:react-hooks/recommended"]
+root: true
+extends: 
+  - "@redhat-cloud-services/eslint-config-redhat-cloud-services"
+  - "plugin:react-hooks/recommended"
+  - "prettier"
```

Comment:
The `prettier` rules should be included via the `eslint-config`[0], or? 


[0] https://github.com/RedHatInsights/frontend-components/blob/master/packages/eslint-config/index.js#L20

Reply:
I applied your suggestion. I hope I understand it correctly.

Reply:
What I meant was, that, since `'@redhat-cloud-services/eslint-config-redhat-cloud-services'` is already included to extend, and that configuration includes `prettier` it should be good to drop those and only have:

```yaml
extends: 
  - "@redhat-cloud-services/eslint-config-redhat-cloud-services"
  - "plugin:react-hooks/recommended"
```

Reply:
ups. fixed